src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -20,13 +20,14 @@
     angulartics2On="click"
     angularticsCategory="Navigation"
     angularticsAction="app_navigation_link_click"
-    [angularticsLabel]="item.label"
+    [angularticsLabel]="item.name"
     *ngFor="let item of menuItems"
     [routerLink]="[item.link]"
+    [class]="activeElement === item.name ? 'highlighted' : ''"
```

Comment:
Did you check whether `[routerLinkActive]="highlighted"` already does the trick? [API Doc](https://angular.io/api/router/RouterLinkActive#routerlinkactive)